src/services/embeddings_service.py

```
@@ -0,0 +1,278 @@
+
+    def get_supported_models(self) -> list[str]:
+        """
+        Get list of supported embedding models.
+
+        Returns:
+            List of model names
+        """
+        if self.config.provider == "openai":
+            return ["text-embedding-ada-002", "text-embedding-3-small", "text-embedding-3-large"]
+        else:
+            return []
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)